code/modules/surgery/surgery_step.dm

```diff
@@ -74,6 +74,14 @@
 	if(tool)
 		speed_mod = tool.toolspeed
 
+	//Bungalow Medical rework
+	if(target!= CONSCIOUS)
```

Comment:
I don't think that's how it works.
I am fairly sure you wanted to use (target.stat != CONSCIOUS) or something like that.

Reply:
I've actually tested that specifically, it does work.

Reply:
It works because "target" mob isn't a "CONSCIOUS" define.
Try making the mob unconscious and it will be the same result, because your check will always be true.

Reply:
got it, fixing